include/drivers/espi.h

```diff
@@ -705,7 +705,7 @@ static inline int z_impl_espi_receive_oob(struct device *dev,
 __syscall int espi_read_flash(struct device *dev,
 			      struct espi_flash_packet pckt);
 
-static inline int z_impl_espi_flash_read(struct device *dev,
+static inline int z_impl_espi_read_flash(struct device *dev,
```

Comment:
Just please clarify if this commit should be part of this PR

Reply:
This is the dependency towards #23796 to allow the build/compile the new functions.
Assuming that PR gets merged first, just rebase will be required here.